src/ray/raylet/local_object_manager.cc

```diff
@@ -217,8 +218,6 @@ bool LocalObjectManager::SpillObjectsOfSize(int64_t num_bytes_to_spill) {
           auto now = absl::GetCurrentTimeNanos();
           RAY_LOG(DEBUG) << "Spilled " << bytes_to_spill << " bytes in "
                          << (now - start_time) / 1e6 << "ms";
-          spilled_bytes_total_ += bytes_to_spill;
```

Comment:
Moved this from the callback to `OnObjectsSpilled` - I don't think we were tracking spilled size updates from functions like `SpillObjects`.